ncm-opennebula/src/main/resources/network_level1.tt

```diff
@@ -19,3 +19,6 @@ IP = "[% interfaces.${interface}.ip %]",
 [%     IF !ignoremac  -%]
 MAC = "[% hardware.cards.nic.${data.key}.hwaddr %]",
 [%     END -%]
+[%     IF system.opennebula.virtio_queues -%]
+VIRTIO_QUEUES = "[% system.opennebula.virtio_queues %]",
```

Comment:
Just checking that this really should be quoted? It looks a little odd as it is a number.